releases/release-1.31/release-notes/maps/pr-126145-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 126145
+releasenote:
+  text: |-
+    Promoted feature-gate `VolumeAttributesClass` to beta (disabled by default). Users need to enable the feature gate and the storage v1beta1 group to use this new feature.
+    Promoted API `VolumeAttributesClass` and `VolumeAttributesClassList` to `storage.k8s.io/v1beta1`.
```

We don't write API kinds in backticks, and we rarely mention that the collection got promoted as well as the object,so:
```suggestion
    Promoted the VolumeAttributesClass API to beta.
```